website/documentation/getting-started/shoots/_index.md

```diff
@@ -0,0 +1,126 @@
+
+Luckily there is a dashboard to get started.
+
+## Basic Configuration Options
+
+![](./images/basic_configurations_1.png)
+
+Every cluster needs a name - after all, it is a Kubernetes resource and therefore unique within a namespace.
+
+The Kubernetes version will be used as a starting point. Once a newer version is available, you can update (but not downgrade, not supported by Kubernetes in general).
+
+The "purpose" defines some configuration (like a monitoring stack or alerting rules) and generally indicates the importance of a cluster.
+
+![](./images/basic_configurations_2.png)
+
+Start by selecting the infrastructure you want to use. The choice will be mapped to a cloud profile that contains provider specific information like the available (actual) OS images, zones and regions or machine types.
+
+Each data plane runs in an infrastructure account owned by the end user. By selecting the infrastructure secret, the user grants Gardener access to the respective account to create / manage resources.
+
+{{% alert color="info"  title="Note" %}}
+Changing the account after the creation of a cluster is not possible. The credentials can be updated with a new key or even user but have to stay within the same account. 
+
+Currently, there is no way to move a single cluster to a different account. A user would rather have to re-create a cluster and migrate workloads by different means.
+{{% /alert %}}
+
+As part of the infrastructure choice, the region for data plane has to be chosen as well. The Gardener scheduler will try to place the control plane on a seed cluster based on a minimal distance strategy. See [Gardener Scheduler](https://github.com/gardener/gardener/blob/master/docs/concepts/scheduler.md) for more details.
+
+![](./images/basic_configurations_3.png)
+
+Up next, the networking provider for the cluster has to be selected (CNI). At the point of writing it is possible to choose between Calico and Cilium. If not specified in the shoot's manifest, default CIDR ranges for nodes, services, and pods will be used.
+
+In order to run any workloads in your cluster, you need nodes. The worker section lets you specify the most important configuration options. For beginners, the machine type is probably the most relevant field, together with the machine image (operating system). 
+
+The machine type is provider-specific and configured in the cloud profile. Check your respective cloud profile if you're missing a machine type. Maybe it is available but not in this region.
+
+The operating system your machines will run is the next thing to choose. Debian-based [GardenLinux](https://github.com/gardenlinux/gardenlinux) is the best choice for most use cases.
+
+Other specifications for the workers include the volume type and size. These settings affect the root disk of each node. Therefore we would always recommend to use an SSD-based type to avoid i/o issues.
```

Comment:
```suggestion
Other specifications for the workers include the volume type and size. These settings affect the root disk of each node. Therefore we would always recommend to use an SSD-based type to avoid i/o issues.

{{% alert color="info"  title="Caveat" %}}
Some machine types (e.g. bare-metal machine types on OpenStack) require you to omit the volume type and volume size settings.
{{% /alert %}}
```